integration-tests/path-prefix/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "gatsby-starter-default",
+  "description": "Gatsby default starter",
+  "version": "1.0.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "dependencies": {
+    "cypress": "^3.1.0",
+    "gatsby": "next",
+    "gatsby-plugin-manifest": "next",
+    "gatsby-plugin-offline": "next",
+    "gatsby-plugin-react-helmet": "next",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2",
+    "react-helmet": "^5.2.0"
+  },
+  "keywords": [
+    "gatsby"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "gatsby build --prefix-paths",
+    "postbuild": "mv public blog && mkdir public && mv blog public/blog",
+    "develop": "gatsby develop",
+    "format": "prettier --write '**/*.js'",
+    "test": "npm-run-all -s build start-server-and-test",
+    "start-server-and-test": "start-server-and-test serve http://localhost:9000/blog cy:run",
+    "serve": "gatsby serve",
+    "cy:open": "cypress open",
+    "cy:run": "cypress run --browser chrome"
```

Comment:
@davidbailey00 what's the value of setting this explicitly to Chrome?

Reply:
We've decided to use Chrome in other Cypress tests due to quirks/possible bugs in the embedded Electron (see the discussion on Discord), so I thought it would be better to make this consistent across all Cypress tests rather than using Chrome for some/Electron for others.

Reply:
Works for me! Thanks for the explanation!